static/js/utils.js

```diff
@@ -13,4 +13,16 @@ export function canonicalizeBoolean (value) {
   } else {
     return false;
   }
+}
+
+/**
+ * returns language extracted from the provided locale
+ * @param {string} locale 
+ * @returns {string} language
+ */
+export function getLanguageFromLocale(locale) {
+  const language = (locale === 'zh-CN' || locale === 'zh-TW')
```

Comment:
This method will also need to be updated to use the new language codes I mentioned above. I'd also let @cea2aj and @oshi97 know you made this method. I think one of them was working on something similar for a different PR.

Reply:
I see that connor made a parse locale function in his SDK pr. Will look over that and probably use that in theme